processor/resourcedetectionprocessor/README.md

```diff
@@ -65,6 +65,7 @@ ec2:
     * aws.ecs.cluster.arn
     * aws.ecs.task.arn
     * aws.ecs.task.family
+    * aws.ecs.task.revision
```

Comment:
Please also add to https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/resource/semantic_conventions/cloud_provider/aws/ecs.md

Reply:
open-telemetry/opentelemetry-specification#1581